ui/components/app/confirm-page-container/confirm-page-container.component.js

```diff
@@ -183,6 +188,7 @@ export default class ConfirmPageContainer extends Component {
             )}
           </PageContainerFooter>
         )}
+        {editingGas && <EditGasPopover onClose={handleCloseEditGas} />}
```

Comment:
When in the SHOW_EIP_1559_UI branch we will no longer trigger the old modal, instead we'll use the Popover directly. editingGas will always be false for the code branch where SHOW_EIP_1559_UI is false.